### ISSUE TITLE
The term "minimum focus indicator area" is confusing

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag#1544]
The current text contains the term "Minimum focus indicator area"
<ul>
<li><strong>Change of contrast:</strong> The pixels making up the minimum focus indicator area achieve at least a 3:1 contrast between their colors in the focused and unfocused states.</li>
        <li><strong>Adjacent contrast:</strong> The pixels in the minimum focus indicator area achieve at least a 3:1 against adjacent colors in the focused control or the minimum focus indicator area has a thickness of at least 2 CSS pixels.</li>	        <li><strong>Adjacent contrast:</strong> The pixels in the minimum focus indicator area achieve at least a 3:1 against adjacent colors in the focused control or the minimum focus indicator area has a thickness of at least 2 CSS pixels.</li></ul>

Its difficult to understand and there is no mention of "minimum focus indicator area" in the understanding. I think we need a defined term.

**minimum focus indicator area:** The part of the focus indicator that is being relied upon to conform to the success criteria. 
**NOTE:** There may be a halo, gradient, or animation on the focus indicator which may not meet contrast minimums, as long as enough of the focus indicator passes the 4 bullet points in the success criteria. There minimum focus indicator area meets the success criteria.

**Comment by mraccess77** [comment]
The first bullet at https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-enhanced is 
Minimum area: The focus indication area is greater than or equal to a 2 CSS pixel solid border around the control.

**Comment by alastc** [comment]
Best to refer to the latest draft: https://w3c.github.io/wcag/guidelines/22/#focus-appearance-minimum

The 1st  & 2nd bullets are:
> * **Minimum area**: The area of the focus indicator is at least as large as as the area of a 1 CSS pixel thick perimeter of the focused control, or the focus indicator has a thickness of at least 8 CSS pixels along the shortest side of the user interface component.
> * **Change of contrast**: The pixels making up the minimum focus indicator area achieve at least a 3:1 contrast between their colors in the focused and unfocused states. 

So the second is building on the first, and "focus indicator" is the defined term.

As discussed, not all the focus indicator needs to have focus, just the min area of that focus indicator.

I don't think an overlapping definition would help, where we have "minimum focus indicator area" and "focus indicator ".

How about re-phrasing the 2nd bullet?
* **Change of contrast**: The pixels making up the **minimum area of the** focus indicator achieve at least a 3:1 contrast between their colors in the focused and unfocused states.

**Comment by alastc** [comment]
Hi @DavidMacDonald, Do you think this is still an issue in the [latest draft](https://w3c.github.io/wcag/understanding/focus-appearance-minimum.html)?

For most people it makes no difference, but if you have a gradient or similar, it is allowing you to discard the non-contrasting pixels so long as there is a min area.

**Comment by alastc** [comment]
Hi David,

Before you joined the meeting today we updated the min-area text (again), it is now:

**Minimum area:** The [link]focus indication area[/link] is either:
- at least as large as the area of a 1 <a>CSS pixel</a> thick perimeter of the unfocused control;
- at least as large as a 4 CSS pixels border along the shortest side, and no thinner than 2 CSS pixels.

With the "focus indicator area" being defined as:
> the pixels that are changed to visually indicate when a user interface component is in a focused state.

There is also a whole section called [minimum area](https://w3c.github.io/wcag/understanding/focus-appearance-minimum.html#size-of-indicator) in the understanding doc. 

Does that help?

**Comment by DavidMacDonald** [comment]
I can live with it... I expect consultants are going to be busy explaining this a lot.

**Comment by alastc** [comment]
Hi David, if you think the suggestion above (or something else) would make it easier, I'm happy to put that to the group.

**Comment by alastc** [comment]
Discussed in the group today, we agreed to keep the current SC text, but work on the understanding doc. I think @rainbreaw and @DavidMacDonald agreed to look at this.

Leaving open for a marker, but not a blocker for putting the SC up for review.

**Comment by joesaiyang** [comment]
I’m looking through WCAG 2.2 SC 2.4.11 - Focus Appearance (Minimum), and part of it states:

> Not fully obscured: The item with focus is not entirely hidden by author-created content.


What if the visible focus indicator is obscuring the “author-created content?” An example would be a link where the visible focus indicator clips parts of the text. Is this addressed in another SC? Or is this not a concern?

<img width="135" alt="a link has a visible focus indicator that slightly clips the first and last letter." src="https://user-images.githubusercontent.com/19971193/108759082-c1fd8800-7500-11eb-972f-aa4e3d36d29a.png">

Or does the fact that `outline-offset` is set to negative 2 violate:

> Minimum area: The focus indicator area is either:
> at least as large as the area of a 1 CSS pixel thick perimeter of the unfocused component

**Comment by LiLoDavis** [comment]
I imagine this suggestion comes too late, but perhaps the requirement could be simplified by providing separate definitions?

## Requirements

- **Change contrast**: The focus indicator includes pixels with a **change contrast** of at least 3:1, and those pixels have an area greater than or equal to the **minimum focus indicator area**.
- **Adjacent contrast**: At least one of the following is true:
  - The focus indicator includes pixels with an **adjacent contrast** of at least 3:1, and those pixels have an area greater than or equal to the **minimum focus indicator area**; or
  - The focus indicator has a thickness of at least 2 CSS pixels.
- **Not fully obscured**: When the component has focus, it is not entirely hidden by author-created content.

## Definitions
**Minimum focus indicator area**: The smaller of two values derived from the area of the component: 
- The area of a 1 CSS pixel thick perimeter of the unfocused component; or
- The area of a 4 CSS pixels border along the shortest side of the unfocused component.

**Change contrast**: The contrast ratio between the colors of (1) pixels in the focus indicator and (2) the same pixels when the component is unfocused.

**Adjacent contrast**: The contrast ratio between the colors of (1) pixels in the focus indicator and (2)  adjacent pixels when the component has focus.

**Comment by alastc** [comment]
@joesaiyang - obscuring the text of the control isn't part of this SC, but I think after a certain point you could raise it under 3.3.2 Labels or Instructions.

@LiLoDavis - that makes it clearer in one way, but they you have to go from the SC text to the definitions, and then back in order to understand the area requirement. I think we might get objections if the sizing wasn't in the main SC text.

**Comment by LiLoDavis** [comment]
Thanks, @alastc - I appreciate your response and get your point.

**Comment by alastc** [comment]
Hi folks,

The SC has been significantly updated ([preview](https://raw.githack.com/w3c/wcag/main/understanding/22/target-size-minimum.html)), and the term "minumum focus area" is not used any more, so closing this issue.
